diff --git a/lib/cjs/fastembed.js b/lib/cjs/fastembed.js
index 16ff4e40a96a2ed9b68e81011256f701bc2e7b82..fc96886d9257c715387298a3450ffcb855766873 100644
--- a/lib/cjs/fastembed.js
+++ b/lib/cjs/fastembed.js
@@ -33,7 +33,7 @@ const https_1 = __importDefault(require("https"));
 const ort = __importStar(require("onnxruntime-node"));
 const path_1 = __importDefault(require("path"));
 const progress_1 = __importDefault(require("progress"));
-const tar_1 = __importDefault(require("tar"));
+const tar_1 = require("tar");
 const hub_1 = require("@huggingface/hub");
 var ExecutionProvider;
 (function (ExecutionProvider) {
@@ -205,7 +205,7 @@ class FlagEmbedding extends Embedding {
     static async decompressToCache(targzPath, cacheDir) {
         // Implementation for decompressing a .tar.gz file to a cache directory
         if (path_1.default.extname(targzPath.toString()) === ".gz") {
-            await tar_1.default.x({
+            await tar_1.x({
                 file: targzPath,
                 // @ts-ignore
                 cwd: cacheDir,
diff --git a/lib/esm/fastembed.js b/lib/esm/fastembed.js
index 620362efd465c6d3ce8c9eb54990c24168b16145..3e1ecd1d482ec42e0836a8417a08a1d2062d73e9 100644
--- a/lib/esm/fastembed.js
+++ b/lib/esm/fastembed.js
@@ -4,7 +4,7 @@ import https from "https";
 import * as ort from "onnxruntime-node";
 import path from "path";
 import Progress from "progress";
-import tar from "tar";
+import * as tar from "tar";
 import { downloadFileToCacheDir } from "@huggingface/hub";
 export var ExecutionProvider;
 (function (ExecutionProvider) {
